Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -2409,7 +2409,7 @@ void LocalDOMWindow::removeAllEventListeners()
 #endif
 
 #if ENABLE(TOUCH_EVENTS)
-    if (RefPtr document = this->document())
+    if (RefPtrAllowingPartiallyDestroyed document = this->document())
```

Comment:
Note that I suggested `RefPtrAllowingPartiallyDestroyed<Document>`. This will NOT build.

Reply:
Seems like, it builds just fine. Should I still change it to `RefPtrAllowingPartiallyDestroyed<Document>`?

Reply:
Yes, please change it. It builds because touch events aren't enabled on bots.

Reply:
Done.